descriptors/ModuleDescriptor-template.json

```diff
@@ -101,7 +101,7 @@
       "description": "Get set structure of a repository"
     },
     {
-      "permissionName": "oia-pmh.all",
+      "permissionName": "oai-pmh.all",
```

Comment:
There are several more such typos in this file. Please fix them as well.

Reply:
Fixed